contents/blog/clickhouse-vs-druid.md

```diff
@@ -0,0 +1,187 @@
+
+- Druid is more complex to set up and maintain
+- Druid is more expensive as it requires more machines and systems
+
+Additionally, Druid is the best solution for applications that need real-time data reporting with precision. For instance, a stock market order-book app may utilize Druid because users expect accurate and aggregate financial data in real-time. 
+
+### Which companies use ClickHouse and why?
+
+- **PostHog:** We [switched from Postgres to Clickhouse](/blog/how-we-turned-clickhouse-into-our-eventmansion) as we scaled and never looked back. By utilizing ClickHouse’s dedicated engines and materialized views, we are able to efficiently return event data to our users with minimal overhead and support billion-event scale companies.
+
+- **Ebay:** Ebay [switched](https://www.youtube.com/watch?v=KI0AqpmcSOk) from Druid to ClickHouse with Apache Zookeeper to dramatically reduce its server costs and simplify its backend. Originally, Ebay’s Druid setup was sprawling and complicated to maintain, racking up serious engineering costs. After switching to ClickHouse, Ebay’s overall infrastructure was reduced by a whopping 90%.
+
+- **Uber:** Uber [moved](https://www.uber.com/blog/logging/) its logging platform to ClickHouse, improving data compression and halving its infrastructure costs. Given that Uber needs to log a lot of geographical information during car rides, ClickHouse was the apt solution for streaming data.
+
+### Which companies use Druid and why?
+
+- **Airbnb:*** Druid enables Airbnb to analyze both historical and real-time metrics, returning data for Airbnb hosts quickly and interactively. Because Airbnb processes hundreds of transactions every minute, Druid enables them to return accurate pricing shifts efficiently.
+
+- **Lyft:** Probably the best example on how Druid and ClickHouse **could** be used interchangeably with additional architecture is Uber’s cousin’s choice in Druid. Lyft utilizes Druid, with some [additional setup](https://www.youtube.com/watch?v=ovZ9iAkQllo), to achieve similar results as Uber has.
+
+- **Rovio:** The makers of everyone’s favorite angry ducklings game, Rovio uses Druid to chop and pivot time-series dashboards filled with data from their millions of players.
+
+## Summary
+
+Druid and ClickHouse are similar solutions solving the same problem with dramatically different designs. ClickHouse champions simplicity and a unified instance; Druid is complex but highly configurable. 
+
+While both offer enormous advantages over traditional databases for columnar data, companies should strongly consider their needs when making choice between the two. 
+
+### ClickHouse
+
+**Pros**
+
+- Low latency
+- Simple to set up, can operate out of a single instance
+- Materialized Views expedite aggregations at the data level
+- Extremely fast query performance on large datasets
```

Comment:
I would probably add a section talking about the variety of table engines on ClickHouse and compare that to what is available over in Druid land
https://clickhouse.com/docs/en/engines/table-engines/

^ Kafka, Materialized Postgres, and others are really interesting here.